wiki/Gameplay/Score/ScoreV1/osu!/en.md

```diff
@@ -1,31 +1,51 @@
----
-needs_cleanup: true
----
-
-# osu! scoring system
+# ScoreV1 (osu!)
 
 *See also: [osu! judgement system](/wiki/Gameplay/Judgement/osu!)*
 
-The score given by each hit circle and end of a slider is calculated with the following formula:-
+In **ScoreV1**, each of the object types in osu! is scored slightly differently.
+However, the total score is a simple sum of points awarded for each individual object in the beatmap.
+
+The rules for scoring each individual object type are outlined in the sections below.
+
+## Hit circles
+
+Each hit circle is assigned a numerical point value using the formula below:
+
+`Score = Hit value * (1 + (Combo multiplier * Difficulty multiplier * Mod multiplier / 25))`
+
+where:
+
+- The *hit value* is the numerical value of the hit circle judgement (50, 100, or 300).
+- The *combo multiplier* is equal to (combo before this hit - 1) or 0, whichever is higher.
+- The *difficulty multiplier* is a value specific to the beatmap being played. The exact formula to compute it is given in the next subsection.
+- The *mod multiplier* is the total multiplier of the set of active mods.
+
+### Difficulty multiplier
+
+The **difficulty multiplier** is equal to an older version of star rating for the beatmap being played. It can be calculated via the following formula:
+
+`Difficulty multiplier = Round((HP Drain + Circle Size + Overall Difficulty + Clamp(Hit object count / Drain time in seconds * 8, 0, 16)) / 38 * 5)`
+
+Note that game modifiers (like Hard Rock or Easy, which change circle size, for instance) do not affect the difficulty multiplier, as the original values of the variables are always used in the formula above regardless of which mods are enabled.
+
+## Sliders
+
+Each slider as a whole produces a 50, 100, or 300 judgement, based on the proportion of slider parts (namely, the slider head, slider end, slider ticks, and slider repeats) hit. This judgement is converted to a score value using the same method that hit circles use.
```

Comment:
may be worth clarifying what the proportions are, same for spinners

Reply:
Reason why I haven't done this is that I wasn't sure if that belongs in here or in the judgement article. I'm fine with including here if you are.

Reply:
ah, you're right

Reply:
Addressed in https://github.com/ppy/osu-wiki/pull/7527